### PR TITLE
1665979 - [Docs] [Admin Guide] "Adding Local Storage" is incomplete

### PR DESCRIPTION
**Pull request by stoobie**
This PR addresses https://bugzilla.redhat.com/show_bug.cgi?id=1665979

This PR applies to the current version of oVirt.

Changes proposed in this pull request:

- Explain local storage and local storage domain
- Adding Local Storage - add step to verify local storage has been added.
- Adding Local Storage - add missing steps.
- QE Validation of Preparing Local Storage

I confirm that this pull request was submitted according to the contribution guidelines: @stoobie

This pull request needs review by: @oVirt/ovirt-documentation, @bennyz

### CODE REVIEW

File: source/documentation/common/storage/proc-Adding_Local_Storage.adoc [hunk 1]
```diff
@@ -13,5 +13,14 @@ Adding local storage to a host places the host in a new data center and cluster.
 . Set the path to your local storage in the text entry field.
 . If applicable, click the *Optimization* tab to configure the memory optimization policy for the new local storage cluster.
 . Click btn:[OK].
+. Click menu:Storage[Domains].
+. Click *New Domain*.
+. Enter a Name for the storage domain.
+. Select the host you defined for as the Data Center
```
Comment (RichardHoch) [inline source/documentation/common/storage/proc-Adding_Local_Storage.adoc:19]: Lines 19, 20, and 21 need a period at their ends.

File: source/documentation/common/storage/proc-Adding_Local_Storage.adoc [hunk 2]
```diff
@@ -13,5 +13,14 @@ Adding local storage to a host places the host in a new data center and cluster.
 . Set the path to your local storage in the text entry field.
```
Comment (RichardHoch) [inline source/documentation/common/storage/proc-Adding_Local_Storage.adoc:13]: Suggestion: Start this sentence with the location unless it is obvious (For example, "In the XYZ manager, ..."

File: source/documentation/common/storage/proc-Adding_Local_Storage.adoc [hunk 3]
```diff
@@ -13,5 +13,14 @@ Adding local storage to a host places the host in a new data center and cluster.
...
 . Click btn:[OK].
+. Click menu:Storage[Domains].
+. Click *New Domain*.
+. Enter a Name for the storage domain.
+. Select the host you defined for as the Data Center
+. Select a Domain Function
+. For Storage Type, select *Local on Host*
+. Select the host to use.
+. Enter the export path to be used for the storage domain. The export path should be in the format of 123.123.0.10:/data (for IPv4), [2001:0:0:0:0:0:0:5db1]:/data (for IPv6), or domain.example.com:/data.
+. Optionally, you can configure the advanced parameters:
```
Comment (RichardHoch) [inline source/documentation/common/storage/proc-Adding_Local_Storage.adoc:24]: Why use a colon here?

**Comment by RichardHoch** [comment]
@stoobie: A few small comments, but otherwise, LGTM!

File: source/documentation/common/storage/proc-Adding_Local_Storage.adoc (hunk 2 above)
Comment (stoobie) [reply source/documentation/common/storage/proc-Adding_Local_Storage.adoc:12]: @RichardHoch I had the same comment when I first started and it wasn't obvious to me. But then I realized that it should be obvious. This is the Administration Guide so it's talking about the Administration Portal. Most or all of the other procedures don't state it.

**Comment by bennyz** [comment]
LGTM from technical standpoint

**Comment by stoobie** [comment]
@bennyz Can you give an official approval?

**Review by bennyz — APPROVED** [review]

**Pull request merged**